### PR TITLE
Add link which redirects to whole organization issues pages

### PR DESCRIPTION
**Pull request by slasher17**
Fixes index

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -42,7 +42,7 @@ <h4 style="text-align:center;font-weight:bold;">We're so happy to meet you! This
           <p>We especially welcome contributions from people from <b>groups underrepresented</b> in free and open source software!</p>
 
           <p>Our community aspires to be a respectful place. Please read and abide by our <a href="https://publiclab.org/conduct">Code of Conduct</a>.</p>
-
+	  <p>To explore issues of entire organization. Check out <a href="https://code.publiclab.org/#r=all">Public Lab</a></p>
```
Comment (jywarren) [inline index.html]:
```suggestion
	  <p>To explore issues across the entire organization, check out <a href="https://code.publiclab.org/#r=all">Public Lab</a></p>
```

**Comment by jywarren** [comment]
Hi, i added one change request, and could you make the PR title more descriptive, like "add link to cross-repository issues"?

Thank you!!!

**Comment by Rishabh570** [comment]
@slasher17 Your PR requires one minor change as suggested above, please take a look :)
Thanks for your contribution :smile:

**Comment by gauravano** [comment]
Please review and merge if you find the text suitable. I added issues after Public Lab. Thanks!

**Comment by jywarren** [comment]
👍 awesome. Thank you!!!

**Pull request merged**